spec/system/dashboard_spec.rb

```diff
@@ -5,4 +5,17 @@
     visit '/good_job'
     expect(page).to have_content 'GoodJob 👍'
   end
+
+  # This requires storing job in database, but rest of the tests is using inline adapter
```

Comment:
When I add this to config/environments/test.rb the unit tests are failing
```
  config.active_job.queue_adapter = :good_job
  config.good_job = {execution_mode: :external}
```

Reply:
I will look into that.

Reply:
@morgoth You can add this:

```
 ActiveJob::Base.queue_adapter = GoodJob::Adapter.new(execution_mode: :external)
 ```
 
The [adapter is reset](https://github.com/bensheldon/good_job/blob/b3dd40dd90a2cc3d8b20c9c40662f9c038b0a353/spec/support/reset_rails_queue_adapter.rb#L1-L7) after every test, so that's should be all you need: